NEWS.md

```diff
@@ -41,6 +41,7 @@ bolded warnings in the full release notes below.
 - **Breaking:** helpops now uses +h instead of +H
 - **Breaking:** sno\_whois and the spy\_ extensions have been removed
 - **Breaking:** Using /wallops now requires the oper:wallops privilege instead of oper:massnotice
+- **Breaking:** Opers wanting to override CALLERID (usermode +g) need oper:message
```

Comment:
I don't think we need a news entry since it already says all privset configs will be broken

Reply:
... I thought I cancelled the comment and then added this review. Oops.